Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -6538,6 +6546,7 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 1DEB927408733DD40010E9CD /* Build configuration list for PBXNativeTarget "TestWebKitAPI" */;
 			buildPhases = (
+				A1508C552DAF1F8000C3E213 /* Process Entitlements */,
```

Comment:
Out of curiosity, do we _need_ to run the script this early, before compilation? I would have expected it to run post-compiling, pre-linking, i.e. between the Sources and Frameworks build phases.

No big deal either way.

Reply:
I'm not sure I totally understand why, but a cycle is introduced when this phase comes after Compile Sources.